### PR TITLE
doc: Secure boot doc update

### PR DESCRIPTION
**Pull request by FrancescoSer**
Updated bootloader.rst

### CODE REVIEW

File: doc/nrf/app_dev/bootloaders_and_dfu/bootloader.rst [hunk 1]
```diff
@@ -62,10 +62,10 @@ There are two implementations currently supported:
 
   See the following image for an abstract representation of the memory layout for an application that uses both an immutable and an upgradable bootloader in its boot chain:
 
-  .. figure:: images/bootloader_memory_layout.svg
+  .. figure:: ../../../../../nrf/app_dev/bootloaders_and_dfu/images/bootloader_memory_layout.svg
```
Comment (FrancescoSer) [inline doc/nrf/app_dev/bootloaders_and_dfu/bootloader.rst:65]: Restore original image paths

File: doc/nrf/app_dev/bootloaders_and_dfu/bootloader.rst [hunk 2]
```diff
@@ -99,7 +99,7 @@ The verification provided by this bootloader is recommended and suitable for all
    You must generate and use your own signing keys while in development and before deploying when using either MCUboot or the |NSIB| as an immutable bootloader.
    See :ref:`ug_fw_update_development_keys` for more information.
 
-The :ref:`bootloader capabilities table <app_bootloaders_support_table>` indicates the bootloaders that are valid for use as immutable bootloaders.
+The bootloader capabilities table lists the bootloaders that you can use as an immutable bootloader.
```
Comment (FrancescoSer) [inline doc/nrf/app_dev/bootloaders_and_dfu/bootloader.rst:102]: restore ref

File: doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst [hunk 3]
```diff
@@ -7,12 +7,12 @@ Secure bootloader chain
    :local:
    :depth: 2
 
-The architecture behind the secure bootloader solutions provided by |NCS| is based on the *chain of trust* concept.
+The nRF Connect SDK (NCS) secure bootloader solutions are built on the *chain of trust* concept.
```
Comment (annwoj) [inline doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst:10]:
```suggestion
The |NCS| (NCS) secure bootloader solutions are built on the *chain of trust* concept.
```

File: doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst [hunk 4]
```diff
@@ -46,7 +46,7 @@ After all of the images in the bootloader chain have been verified successfully,
 Architecture
 ************
 
-There are two implementations currently supported:
+The nRF Connect SDK (NCS) currently supports two implementations:
```
Comment (annwoj) [inline doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst:49]:
```suggestion
The |NCS| currently supports two implementations:
```

File: doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst [hunk 5]
```diff
@@ -106,7 +106,7 @@ If the verification fails, the boot process stops.
 This way, the immutable bootloader can guarantee that the next image in the boot sequence will not start up if it has been tampered with in any way.
 For example, if an attacker attempts to take over the device by altering the firmware, the device will not boot, and thus not run the infected code.
 
-The immutable bootloader is flash-locked and cannot be modified or deleted without erasing the entire device.
+The immutable bootloader is locked to the flash memory and can't be modified or deleted without erasing the entire device.
```
Comment (annwoj) [inline doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst:109]:
```suggestion
The immutable bootloader is locked to the flash memory and cannot be modified or deleted without erasing the entire device.
```

File: doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst [hunk 6]
```diff
@@ -150,7 +150,7 @@ Pre-signed variants
 When programming an upgradable bootloader, the build system can automatically generate pre-signed variants of the image verified by the |NSIB|.
 The upgradable bootloader does not use pre-signed variants for updating the application.
```
Comment (annwoj) [inline doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst:151]:
```suggestion
The upgradable bootloader does not use pre-signed variants to update the application.
```

File: doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst [hunk 7]
```diff
@@ -150,7 +150,7 @@ Pre-signed variants
...
 
-You must build with pre-signed variants when building upgrade images for the image that follows the |NSIB| in the boot chain, such as the upgradable bootloader or the application.
+When building upgrade images for the image following the |NSIB| in the boot chain, like the upgradable bootloader or the application, you must build with pre-signed variants.
```
Comment (annwoj) [inline doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst:153]:
```suggestion
When building upgrade images for the image following the |NSIB| in the boot chain, like the upgradable bootloader or application, you must build with pre-signed variants.
```

**Review by greg-fer — APPROVED** [review]

**Review by annakielar — APPROVED** [review]

File: doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst [hunk 8]
```diff
@@ -7,12 +7,12 @@ Secure bootloader chain
    :local:
    :depth: 2
 
-The architecture behind the secure bootloader solutions provided by |NCS| is based on the *chain of trust* concept.
+The |NCS|| (NCS) secure bootloader solutions are built on the *chain of trust* concept.
```
Comment (annakielar) [inline doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst:10]:
```suggestion
The |NCS| (NCS) secure bootloader solutions are built on the *chain of trust* concept.
```

**Review by annwoj — APPROVED** [review]

**Pull request merged**